src/commons/authentication/mod.rs

```diff
@@ -29,17 +34,28 @@ pub(crate) const SECRET_BASE_PATH: &str = "/stackable/secrets";
 )]
 #[serde(rename_all = "camelCase")]
 pub struct AuthenticationClassSpec {
-    /// Provider used for authentication like LDAP or Kerberos
+    /// Provider used for authentication like LDAP or Kerberos.
     pub provider: AuthenticationClassProvider,
 }
 
 #[derive(Clone, Debug, Deserialize, Display, Eq, JsonSchema, PartialEq, Serialize)]
 #[serde(rename_all = "camelCase")]
 #[allow(clippy::large_enum_variant)]
 pub enum AuthenticationClassProvider {
+    /// The [static provider](https://DOCS_BASE_URL_PLACEHOLDER/concepts/authentication#_static) is used to configure a
+    /// static set of users, identified by username and password.
     Static(static_::AuthenticationProvider),
+
+    /// The [LDAP provider](DOCS_BASE_URL_PLACEHOLDER/concepts/authentication#_ldap).
+    /// There is also the ["Authentication with LDAP" tutorial](DOCS_BASE_URL_PLACEHOLDER/tutorials/authentication_with_openldap)
+    /// where you can learn to configure Superset and Trino with OpenLDAP.
     Ldap(ldap::AuthenticationProvider),
+
+    /// The OIDC provider can be used to configure OpenID connect.
```

Comment:
I think thats treated as a name.
```suggestion
    /// The OIDC provider can be used to configure OpenID Connect.
```